onebot/plugins/botui.py

```diff
@@ -132,6 +132,15 @@ def quote(self, mask, target, args):
         self.bot.send(cmd)
 
     @command(permission='admin', show_in_help_list=False)
+    def load(self, mask, target, args):
```

Comment:
I added some error handling to limit it to plugins that have already been loaded. I believe it's possible to load new plugins, but it's not really what the `reload()` method was intended for. I added this mainly for debugging and changing settings.